recipes/editorconfig-core

```diff
@@ -1,3 +1,3 @@
-(editorconfig-core :repo "editorconfig/editorconfig-emacs"
+(editorconfig-core :repo "10sr/editorconfig-core-emacslisp"
```

Comment:
Thanks. I have reverted these changes and will fix this on the Emacsmirror instead.